src/main/resources/assets/api-docs.yaml

```diff
@@ -108,6 +108,8 @@ paths:
           description: Deprecated endpoint. Use  `GET /api/tdr/{identifier}` instead.
   /api/daa/{dacId}:
     $ref: 'paths/daaById.yaml'
+  /api/daa:
+    $ref: 'paths/findAllDaa.yaml'
```

Comment:
nit: naming. The path yaml should reflect the path, not necessarily the operation. For example, we could call GET or DELETE (a bad idea in this case, but a potential) and they would both be defined in the same endpoint, just different methods. Another more concrete example for `/api/daa/{dacId}` ... we could possibly call GET, PUT, DELETE on that endpoint so `daaById.yaml` would contain docs for all of those and therefore the naming makes more sense.

Perhaps `findAllDaa.yaml` -> `daa.yaml`